index.html

```diff
@@ -0,0 +1,82 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <link rel =" stylesheet" href="Style.css" />
+  <title>My CV</title>
+</head>
+<body class="body_class">
+  <header>
+    <h1 class="h1_class">CV</h1>
+  </header>
+  <nav>
+      <ul class = "nav_class">
```

Comment:
I think, that it is not necessary to use "class" in name of the class, as it is already clear, that this is the class :)